### PR TITLE
Rust is Good Programming Langage.

### PR DESCRIPTION
**Pull request by k-masatany**

### CODE REVIEW

File: unko [hunk 1]
```diff
@@ -1,20 +0,0 @@
-#!/bin/sh
```
Comment (nmbakfm) [inline unko]: Don't remove original code!

Comment (k-masatany) [reply unko]: Oops! Sory!

**Comment by k-masatany** [comment]
Please merge my unko into your unko.

**Comment by nmbakfm** [comment]
LGTM

**Pull request merged**